mm2src/trezor/src/utxo/sign_utxo.rs

```diff
@@ -0,0 +1,228 @@
+use crate::proto::messages_bitcoin as proto_bitcoin;
+use crate::utxo::unsigned_tx::UnsignedUtxoTx;
+use crate::utxo::Signature;
+use crate::{TrezorClient, TrezorError, TrezorResponse, TrezorResult};
+use common::log::{debug, info};
+use common::mm_error::prelude::*;
+
+const NO_DETAILS_ERROR: &str = "'TxRequest::details' is expected to be set";
```

Comment:
I think it will be overhead to add `TrezorSignUtxoError` that extends `TrezorError` with these `NO_DETAILS_ERROR`, `NO_REQUEST_INDEX_ERROR` etc, that relate to the protocol only.

Reply:
Ok, I have rechecked the code and it seems acceptable in this case. Thanks for clarification!